packages/app-admin/src/types.ts

```diff
@@ -53,15 +59,33 @@ export type MenuContentSectionPlugin = Plugin & {
     render(props: { Section: typeof Section; Item: typeof Item }): React.ReactNode;
 };
 
+/**
+ * Enables adding custom header elements to the left side of the top bar.
+ * @see https://docs.webiny.com/docs/webiny-apps/admin/development/plugins-reference/app#header-left
+ */
 export type HeaderLeftPlugin = Plugin & {
+    type: "header-left";
+    description: "Enables adding custom header elements to the left side of the top bar.";
```

Comment:
Remove the `description` in the actual TS type.

Reply:
The changes has been effected